crm-plugin/testcases/activity.test.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+/* jshint node:true */
+
+var Testcase = require("./common/testcase");
+
+var testcase = new Testcase();
+function State() {
+  /*this method calls all testcases
+   this method calls all testcase methods
+  */
+  this.index = async () => {
+    var find = await this.emptyTestcase("find");
+    var create = await this.emptyTestcase("create");
+    var deleteMethod = await this.emptyTestcase("delete");
+  };
+
+  /* this method calls all testcases for empty params check */
+  this.emptyTestcase = async (method) => {
+    let methodParams = {};
+    //this switch case needs to handle all methods of state controller
+    switch (method) {
+      case "find":
+        methodParams = {
+          originalUrl: "/crm-plugin/activities",
+          query: {},
```

Comment:
handle badrequest method

Reply:
also handle other methods for required and correct params

Reply:
done